tests/test_modeling_common.py

```diff
@@ -178,7 +178,6 @@ def _config_zero_init(config):
 
 @require_torch
 class ModelTesterMixin:
-
```

Comment:
Could you revert all those line deletions? It's not really linked to the PR.

Reply:
Actually all of the line deletion were made by `$ make style` otherwise I am getting a `check_code_quality` error, is this a bug?or should I revert those deletions ?

Reply:
You should make sure you have the proper version of `black`, we use 22.3